ruby_on_rails/app_initialisation.md

```diff
@@ -70,10 +70,12 @@ They are always idempotent (runnable multiple times).
 ### ENV variables with Figaro
 
 * Add `figaro` to Gemfile. Check the [gem homepage](https://github.com/laserlemon/figaro) to see how to install the gem
-(usually `bundle exec figaro install` is enough). Delete the newly created file `config/application.yml`.
+(usually `bundle exec figaro install` is enough). Delete the newly created file `config/application.yml`...
 * and create `config/application.example.yml` where you will specify the only environment variable you need for now:
   `SECRET_KEY_BASE`.
-* Add the following section to your `bin/setup` script so that the application.yml is created when the project is setup:
+* Going forward we will only push the `config/application.example.yml` file to the repository in order to protect our env variables.
+* Add application.yml to .gitignore
+* Add the following section to your `bin/setup` script so that the `application.yml` is created from the `application.example.yml` when the project is setup locally:
```

Comment:
I felt this gem could do with a little more explanation for people who aren't familiar (me)